sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -135,7 +136,7 @@ public async Task StartRecognizeReceiptsPopulatesExtractedReceipt(bool useStream
             if (useStream)
             {
                 using var stream = new FileStream(TestEnvironment.ReceiptPath, FileMode.Open);
-                operation = await client.StartRecognizeReceiptsAsync(stream, default);
+                operation = await client.StartRecognizeReceiptsAsync(stream, ContentType.Jpeg);
```

Comment:
Test wouldn't pass if I left it as default. Could you run it in your machine, @annelo-msft? Not sure why CI is not complaining (we need to take a look at that later).

Reply:
Let's track this as part of the auto-detect content type work.